src/main/java/spoon/support/compiler/jdt/ParentExiter.java

```diff
@@ -37,63 +37,63 @@
 import org.eclipse.jdt.internal.compiler.ast.TypeReference;
```

Comment:
In fact the file changed a lot because my first patch was in this file which caused  the checkstyle mess with imports... Not sure how to remove file from the PR properly.